src/services/importService.ts

```diff
@@ -61,33 +65,61 @@ export default class ImportService implements IImportService {
             },
             autoSave: true,
         };
+
+        if (originalProject.visitedFrames.every((item) => typeof item === "number")) {
+            // then this is a video project
+            convertedProject.lastVisitedAssetId = "dummyId";
```

Comment:
Why setting `lastVisitedAssetId` to this value?  This is an optional property and if we don't have a real value to set it to we shouldn't bother setting it.